applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -112,6 +112,10 @@ config BT_VCS
 	bool
 	default y
 
+config BT_PACS_SRC_CONTEXT
+	hex
+	default 0x0005
```

Comment:
What is 0x0005? I cant find it in this list:
![image](https://user-images.githubusercontent.com/89020256/179157656-23bf8735-c6dc-4393-b05d-54339427ebbe.png)

Reply:
I guess it's Unspecified + Media?

Reply:
Yes, 0x0005 is Unspecific + Media.
Since Unspecific should be the mandatory.